src/pages/AppBody.tsx

```diff
@@ -11,6 +12,7 @@ export const BodyWrapper = styled.main<{ margin?: string; maxWidth?: string }>`
     0px 24px 32px rgba(0, 0, 0, 0.01);
   border-radius: 24px;
   margin-top: 1rem;
+  z-index: ${Z_INDEX.content};
```

Comment:
i think it would be cleaner to put `z-index` inside the `theme` object similar to color or media-queries so we don't have to import `Z_INDEX` from `theme` and instead be able to do `z-index: ${({theme}) => theme.zIndexContent`

Reply:
see https://github.com/Uniswap/uniswap-interface/blob/56f8a7443cfa19c329f4d1e557dd8d4713dd76e5/src/theme/index.tsx#L107

Reply:
so i'm with you on color for sure, but i wonder when you would want media queries or z-index to be dynamic with the theme. using enums means they can be compiled into static values vs a function can just add more JS even if the value never changes

Reply:
Really just for developer experience and consistency with how we do media queries. I hadn't really thought about compiled vs. runtime.

Probably makes sense to keep it static.